airbyte-cdk/python/airbyte_cdk/sources/declarative/auth/token.py

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+import base64
+from typing import Union
+
+from airbyte_cdk.sources.declarative.interpolation.interpolated_string import InterpolatedString
+from airbyte_cdk.sources.declarative.types import Config
+from airbyte_cdk.sources.streams.http.requests_native_auth.abtract_token import AbstractHeaderAuthenticator
+
+
+class ApiKeyAuthenticator(AbstractHeaderAuthenticator):
+    """
+    ApiKeyAuth sets a request header on the HTTP requests sent
```

Comment:
for the docstrings i recommend adding an example of what the resulting header lookks like

Reply:
added examples for the 3 authenticators